src/herder/HerderSCPDriver.cpp

```diff
@@ -723,44 +694,34 @@ HerderSCPDriver::nominatingValue(uint64_t slotIndex, Value const& value)
     if (Logging::logDebug("Herder"))
         CLOG(DEBUG, "Herder") << "nominatingValue i:" << slotIndex
                               << " v: " << getValueString(value);
-
-    if (!value.empty())
-    {
-        mSCPMetrics.mNominatingValue.Mark();
-    }
 }
 
 void
 HerderSCPDriver::updatedCandidateValue(uint64_t slotIndex, Value const& value)
 {
-    mSCPMetrics.mUpdatedCandidate.Mark();
 }
 
 void
 HerderSCPDriver::startedBallotProtocol(uint64_t slotIndex,
                                        SCPBallot const& ballot)
 {
     recordSCPEvent(slotIndex, false);
-    mSCPMetrics.mStartBallotProtocol.Mark();
 }
 void
 HerderSCPDriver::acceptedBallotPrepared(uint64_t slotIndex,
                                         SCPBallot const& ballot)
 {
```

Comment:
should we keep those virtual empty methods just for the sake of having extension points?

Reply:
you mean in the main interface? Yes, seems fine to keep them for now as those are quite special events